src/docs/Doxyfile.9

```diff
@@ -2373,7 +2373,7 @@ EXTERNAL_PAGES         = YES
 # powerful graphs.
 # The default value is: YES.
 
-CLASS_DIAGRAMS         = NO
+# CLASS_DIAGRAMS         = NO
```

Comment:
According to v1.9.3 [changelog](https://www.doxygen.nl/manual/changelog.html) "CLASS_DIAGRAM has become obsolete. The same functionality can now be achieved using HAVE_DOT and CLASS_GRAPH".

Commenting out this line is a simple way to address the version compatibility problem.

Reply:
Is there any benefit to leaving this as a comment rather than removing?

Reply:
The commented-out config option will still be valid for some earlier versions of 1.9.x - keep it informational for them.